web-admin/vite.config.ts.timestamp-1683100021671.mjs

```diff
@@ -0,0 +1,23 @@
+// vite.config.ts
+import { sveltekit } from "file:///Users/rakeshrilldata/Workspace/rill-developer/node_modules/@sveltejs/kit/src/exports/vite/index.js";
```

Comment:
What is this?

Reply:
removed this file, not sure how this is added